tests/Rector/v9/v0/ReplaceExtKeyWithExtensionKey/Fixture/ext_tables.php.inc

```diff
@@ -3,6 +3,8 @@
 use TYPO3\CMS\Extbase\Utility\ExtensionUtility;
 
 if (TYPO3_MODE === 'BE') {
+    // must be defined
+    $_EXTKEY = '___';
 
     /**
```

Comment:
Test files are autoloaded, so any missing variable will be reported as fatal error (PHP 8 here :))

This little line will make sure the code is valid.

Another practise we use in Rector core is to wrap code to a class and method `run` by default.
Code works, is autoloaded but is not run.